.github/ISSUE_TEMPLATE/question.yaml

```diff
@@ -0,0 +1,34 @@
+name: Question
+description: Ask a question which is not a bug or a feature request
+labels: "question"
+body:
+  - type: checkboxes
+  attributes:
+    label: Guidelines
+    description: Please ensure you've completed all of the following.
+    options:
+        - label: I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a question that matches the one i want to file, without success.
+          required: true
+- type: checkboxes
```

Comment:
Is it possible to make at least one checkbox required?

Reply:
Good catch :+1: